File: CS/MainForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars;

namespace LargeRibbonButton
{
    public partial class MainForm : Form
    {
        public MainForm()
        {
            InitializeComponent();
        }

        private void radioGroupImageSize_EditValueChanged(object sender, EventArgs e)
        {
            customRibbonControlMain.Hide();
            customRibbonControlMain.LargeImageSize = (CustomRibbonControl.LargeImageSizeCollection)radioGroupImageSize.EditValue;
            customRibbonControlMain.Show();            
        }
    }
}

File: VB/MainForm.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars

Namespace LargeRibbonButton
    Partial Public Class MainForm
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub radioGroupImageSize_EditValueChanged(ByVal sender As Object, ByVal e As EventArgs) Handles radioGroupImageSize.EditValueChanged
            customRibbonControlMain.Hide()
            customRibbonControlMain.LargeImageSize = CType(radioGroupImageSize.EditValue, CustomRibbonControl.LargeImageSizeCollection)
            customRibbonControlMain.Show()
        End Sub
    End Class
End Namespace
